src/components/ArticlesCard.jsx

```diff
@@ -17,7 +17,8 @@ export default function ArticlesCard() {
   }
```

Comment:
This is always going to be false, as any array value is truthy. So the user is not seeing your loading message at any point. I'd suggest holding a boolean loading value in state. isLoading for instance. Then you can directly tie the loading state with the useEffect and the api call.